content/en/developers/metrics/types.md

```diff
@@ -2,298 +2,327 @@
+## Overview
+
+Each metric submitted to Datadog should have a metric type. A metric’s type affects how the metric values are displayed when queried as well as the associated graphing possibilities within Datadog using additional [modifiers][1] and [functions][2]. A metric’s type is displayed on the details side panel for the given metric on the [Metrics Summary page][3].
+
+**Note**: Changing the metric type in this details side panel can potentially change metric behavior in all existing visualizations and monitors; potentially rendering historical data as nonsensical.
 
 The following metric submission types are accepted:
 
-* [COUNT](?tab=count#metric-type-definition)
-* [RATE](?tab=rate#metric-type-definition)
-* [GAUGE](?tab=gauge#metric-type-definition)
-* [HISTOGRAM](?tab=histogram#metric-type-definition)
-* [DISTRIBUTION](?tab=distribution#metric-type-definition)
-* [SET](?tab=set#metric-type-definition)
-* [TIMERS](?tab=timers#metric-type-definition)
+- [COUNT](?tab=count#metric-types)
+- [RATE](?tab=rate#metric-types)
+- [GAUGE](?tab=gauge#metric-types)
+- [HISTOGRAM](?tab=hisogram#metric-types)
+- [DISTRIBUTION](?tab=distribution#metric-types)
+
+These different metric submission types are mapped to 4 in-app metric types found within the Datadog web application:
 
-Once a metric is submitted, there are 3 metric storage types within the Datadog web application:
+- COUNT
+- RATE
+- GAUGE
+- DISTRIBUTION
 
-* `COUNT`
-* `RATE`
-* `GAUGE`
+## Submission vs. In-App type
 
-Refer to the [Metric Type Definition](#metric-type-definition) section below to learn more about the different metrics types available. Also, refer to the [Submission Types and Datadog In-App Types Section](#submission-types-and-datadog-in-app-types) to see how each metric type behaves between its submission and its storage within Datadog.
+Metrics are submitted to Datadog in three main ways:
 
-## Metric type definition
+- [Custom Agent check][4]
+- [DogStatsD][5]
+- [Datadog’s HTTP API][6]
 
-To better understand the different metrics types, what they represent, and how to modify them within Datadog, consider the following example:
+The majority of data Datadog receives is submitted via the Agent (either Agent check or DogStatsD). For these submission methods, a metric’s type determines how multiple values collected on an Agent in [a flush time interval of 10s][7] are aggregated. The Agent combines these values into a single representative metric value for that interval. This combined value is stored with a single timestamp in Datadog.
```

Comment:
The majority of data Datadog receives is submitted via the Agent (either Agent check or DogStatsD). For these submission methods, a metric’s type determines how multiple values collected on an Agent in [a flush time interval][7] are aggregated. The Agent combines these values into a single representative metric value for that interval. This combined value is stored with a single timestamp in Datadog.